File: CPP/palindrome_number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0){return false;}
        std::string temp1 = std::to_string(x);
        size_t size = temp1.size();
        for(size_t i = 0; i < size; i++){
            if(temp1[i]!=temp1[size - 1 - i]){
                return false;
            }
        }
    return true;
    }
};

File: Python/palindrome_number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        temp = x
        rev = 0
        if (x<0): return 0
        while(x):
            rev = rev*10 + (x%10)
            x/=10
        return rev==temp
